x/stablestake/types/expected_keepers.go

```diff
@@ -8,6 +8,7 @@ import (
 // AccountKeeper defines the expected account keeper used for simulations (noalias)
 type AccountKeeper interface {
 	GetAccount(ctx sdk.Context, addr sdk.AccAddress) types.AccountI
+	GetModuleAddress(moduleName string) sdk.AccAddress
```

Comment:
This won't be required if it uses authtypes.NewModuleAddress

Reply:
Got it, thanks.

Reply:
Fixed